cmd/program/program.go

```diff
@@ -172,6 +172,12 @@ func (p *Project) CreateMainFile() error {
 		return err
 	}
 
+	err = utils.GoFmt(projectPath)
+	if err != nil {
+		log.Printf("Could not gofmt in new project %v\n", err)
+		cobra.CheckErr(err)
+	}
+
```

Comment:
Correct me if i am wrong but wouldnt this always fail for users who do not have gofmt installed on their computer/env?

Reply:
Yes and no.
It would fail if gofmt is not installed, but when installing go for the first time gofmt should be installed as well.
If i look in a tar file here: https://go.dev/dl/ I can see that in the go/bin file there's a go and gofmt binary present.

Reply:
Good point. 
I think I am OK to add this then!